test/wasmfs/set_root_backend.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+// This file allows users to easily set a particular backend as the root.
```

Comment:
How about ".... allows tests to run using a particular backend as the root."

Reply:
7bbe6953518c3e2f49f9a4033bcf4b073f548d84